Documentation/devel/pod-lifecycle.md

```diff
@@ -65,17 +71,23 @@ Should #Prepare fail or be interrupted, `$var/prepare/$uuid` will be left in an
 
 `rkt run` and `rkt run-prepared` both arrive here with the pod at `$var/run/$uuid` while holding the exclusive lock.
 
-The pod is then executed while holding this lock.  It is required that the stage 1 `coreos.com/rkt/stage1/run` entrypoint keep the file descriptor representing the exclusive lock open for the lifetime of the pod's process.  All this requires is that the stage 1 implementation not close the inherited file descriptor.  This is facilitated by supplying stage 1 its number in the RKT_LOCK_FD environment variable.
+The pod is then executed while holding this lock.
+It is required that the stage 1 `coreos.com/rkt/stage1/run` entrypoint keep the file descriptor representing the exclusive lock open for the lifetime of the pod's process.
+All this requires is that the stage 1 implementation not close the inherited file descriptor.
+This is facilitated by supplying stage 1 its number in the RKT_LOCK_FD environment variable.
 
 What follows applies equally to `rkt run` and `rkt run-prepared`.
 
 ## Death / exit
 
-A pod is considered exited if a shared lock can be acquired on `$var/run/$uuid`.  Upon exit of a pod's process, the exclusive lock acquired before entering the #Run phase becomes released by the kernel.
+A pod is considered exited if a shared lock can be acquired on `$var/run/$uuid`.
+Upon exit of a pod's process, the exclusive lock acquired before entering the #Run phase becomes released by the kernel.
```

Comment:
These aren't inline html anchors. They're just weird hashtags. Change to `monospace` type for literal string, eg, s/#Run/`Run`/